<launch>

  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find pandora_control)/config/pandora_control.yaml" command="load"/>

  <!-- load the controllers -->
  <node name="kinect_controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
	output="screen" args="kinect_pitch_joint_position_controller
						  kinect_yaw_joint_position_controller"/>

  <node name="kinect_control_node" pkg="pandora_control" type="kinect_control_node" 
		respawn="false" output="screen" />
  
</launch>
